src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -53,6 +54,10 @@
  * Provides shortcuts for HTTP-related features in controllers.
  *
  * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * The following annotation is necessary for compatibility with doctrine/annotations:^2.
```

Comment:
This is the only instance of `@IgnoreAnnotation` in the codebase, so I added that for future developers which may be surprised, and removing it won't break the tests so it may be removed by mistake.

Reply:
please see my comment below to actually investigate the root cause of why this fails with doctrine/annotations 2.0 but not with doctrine/annotations 1.14 as this either indicates an undocumented BC break or a bug.